src/Client/StoreWebhookClient.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NodelessIO\Client;
+
+use NodelessIO\Response\StoreWebhookResponse;
+
+class StoreWebhookClient extends AbstractClient
+{
+    public function allWebhooks(
+        string $storeId
+    ): StoreWebhookResponse {
+        $url = $this->getApiUrl() . 'store/' . urlencode($storeId) . '/webhooks';
```

Comment:
Good catch, i updated the app to make this singular, please update all method endpoints to /webhook

Reply:
Updated